samples/unit-tests/annotations/annotations-shapes/demo.details

```diff
@@ -1,6 +1,7 @@
 ---
  resources:
+   - https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js
```

Comment:
Why is this needed?

Reply:
Thanks for noticing. A sloppy eye on my end. I was playing with karma and forgot to undo this one.